core/dbt/include/global_project/macros/adapters/timestamps.sql

```diff
@@ -0,0 +1,44 @@
+{%- macro current_timestamp() -%}
+    {{ adapter.dispatch('current_timestamp', 'dbt')() }}
+{%- endmacro -%}
+
+{% macro default__current_timestamp() %}
+    current_timestamp
+{% endmacro %}
+
+{%- macro current_timestamp_in_utc() -%}
+    {{ adapter.dispatch('current_timestamp_in_utc', 'dbt')() }}
+{%- endmacro -%}
+
+{% macro default__current_timestamp_in_utc() %}
+    {{ default__current_timestamp() }} at time zone 'utc'
+{% endmacro %}
+
+{%- macro snapshot_get_time() -%}
+    {{ adapter.dispatch('snapshot_get_time', 'dbt')() }}
+{%- endmacro -%}
+
+{% macro default__snapshot_get_time() %}
+    {{ current_timestamp() }}
+{% endmacro %}
+
+{%- macro convert_timezone(timestamp,  target_tz, source_tz) -%}
+    {%- if not target_tz is string -%}
+        {{ exceptions.raise_compiler_error("'target_tz' must be a string") }}
+    {%- else -%}
+        {{ adapter.dispatch('convert_timezone', 'dbt') (column, target_tz, source_tz) }}
+    {%- endif -%}
+
+{%- endmacro -%}
+
+{%- macro default__convert_timezone(column, target_tz, source_tz) -%}
+    {%- if not source_tz -%}
+        {{ column }} at time zone '{{ target_tz }}'
+    {%- else -%}
+        {{ column }} at time zone '{{ source_tz }}' at time zone '{{ target_tz }}'
+    {%- endif -%}
+{%- endmacro -%}
+
+{%- macro now(tz=None) -%}
+{{ convert_timezone(current_timestamp_in_utc(), tz) }}
+{%- endmacro -%}
```

Comment:
Is the idea here to offer an alias for `{{ convert_timezone(tz, current_timestamp_in_utc()) }}`, in keeping with Postgres-y syntax?

Reply:
I couldn't find `NOW` within the SQL standard, and implementation of it across a wide variety of databases is hit-and-miss.

Summary across the database adapters maintained by dbt Labs:
- It doesn't look like either Snowflake or BigQuery implement a `now` function.
- Redshift [deprecated their implementation](https://docs.aws.amazon.com/redshift/latest/dg/c_SQL_functions_leader_node_only.html#:~:text=The%20following%20leader%2Dnode%20only%20functions%20are%20deprecated) of `now`.
- Postgres has a built-in function named `now()` which is an alias for [`transaction_timestamp`](https://www.postgresql.org/docs/current/functions-datetime.html#FUNCTIONS-DATETIME-CURRENT:~:text=now%20(%20)%20%E2%86%92%20timestamp%20with%20time%20zone). It returns the "current date and time (start of current transaction)". The return type is equivalent to Snowflake's `timestamp_ltz`.
- In Spark and Databricks, `now` is an alias for `current_timestamp` (with a return type also equivalent to Snowflake's `timestamp_ltz`).

Reply:
I think it's nice syntactic sugar/short hand but not necessary